Formula/julia.rb

```diff
@@ -80,8 +80,14 @@ def install
       LIBLAPACKNAME=libopenblas
       USE_BLAS64=0
       PYTHON=python3
-      MACOSX_VERSION_MIN=#{MacOS.version}
     ]
+    on_macos { args << "MACOSX_VERSION_MIN=#{MacOS.version}" }
```

Comment:
This actually has no impact on the Linux build, so there's no need to wrap this in `on_macos`.